amazonia-1/amazonia-1-tiler-example.ipynb

```diff
@@ -0,0 +1,721 @@
+{
```

Comment:
<p>should we add a second example showing how to use the assets to request a different band combination, maybe even an NDVI ... expression": "(B4-B3)/(B4+B3)"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/developmentseed/asdi-examples/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>